### ISSUE TITLE
Measure with Boolean multiplication cause format string to be disabled.

### ISSUE THREAD

**Issue by SharmaAntriksh** [TabularEditor/TabularEditor3#113]
If I create a measure like the following one where I doing an array operation (think Excel's SUMPRODUCT) in TE3 then in PBI the formatting options are disabled.
![image](https://user-images.githubusercontent.com/73054674/123847232-f90efa80-d933-11eb-9163-c62cd7309a01.png)

But If I create the same measure in Power BI then the options are available.
![image](https://user-images.githubusercontent.com/73054674/123847419-32e00100-d934-11eb-97ee-e0fe17848a8e.png)

**Comment by otykier** [comment]
Does this only happen to this particular measure? My assumption would be that this is the default behavior of Power BI Desktop whenever a measure is added through an external tool. As such, there is not much I can do about it on the Tabular Editor side. Remember that you can set the format string property through Tabular Editor as well.

**Comment by SharmaAntriksh** [comment]
Yup, this is only happening with this kind of measure, where I am filtering inside SUMX without using and explitcit function. Not a problem at all but thought I should raise a ticket if it can help in perfecting the tool. :)
